freqtrade/exchange/exchange.py

```diff
@@ -903,7 +958,9 @@ def fetch_order(self, order_id: str, pair: str) -> Dict:
         if self._config['dry_run']:
             return self.fetch_dry_run_order(order_id)
         try:
-            order = self._api.fetch_order(order_id, pair)
+            order = self._order_contracts_to_amount(
+                self._api.fetch_order(order_id, pair)
+            )
             self._log_exchange_response('fetch_order', order)
```

Comment:
While the end result will be identical - can we change this to the following (throughout) the exchange class:

```suggestion
            order = self._api.fetch_order(order_id, pair)
            self._log_exchange_response('fetch_order', order)
            order = self._order_contracts_to_amount(order)
```

This will make sure that
a) we're logging the original response - not something we've attempted to modify
b) we can easily add debug-points to the response directly, allowing for easier debugging.